collections-arrays/ArrayPool/ArrayPool/Methods/GetArrayPool.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Buffers;
+
+namespace ArrayPool;
+
+public partial class Methods
+{
+    public static ArrayPool<T> GetDefaultArrayPool<T>()
+    {
+        var arrayPool = ArrayPool<T>.Shared;
+
+        return arrayPool;
+    }
```

Comment:
This could be an expression body statement

Reply:
Fixed!